Client/Help/src/app/components/content-navigation/content-navigation.component.html

```diff
@@ -0,0 +1,6 @@
+<main class="flex w-screen justify-around">
+    <section [routerLink]="['/feed-content']" [ngClass]="{'border-black border-b-2 pb-1': nav ==='feed','text-gray-300': nav !=='feed'}" class="text-sm">Latest</section>
+    <section [routerLink]="['/message-content']" [ngClass]="{'border-black border-b-2 pb-1': nav ==='messages','text-gray-300': nav !=='messages'}" class="text-sm">Messages</section>
+    <section [routerLink]="['/connections-content']" [ngClass]="{'border-black border-b-2 pb-1': nav ==='connections','text-gray-300': nav !=='connections'}" class="text-sm">Connections</section>
+    <section [ngClass]="{'border-black border-b-2 pb-1': nav ==='support','text-gray-300': nav !=='support'}" class="text-sm">Support</section>
```

Comment:
These should really be anchor elements instead. Semantically, sections do not make sense. The impact of using sections can be seen when mousing over the nav items and not seeing the cursor change. I'm confident a screen reader would have no idea these are navigation links either.

Reply:
You are correct. This is not semantic and I have no idea what I was thinking. Fixed.